bin/deploy.sh

```diff
@@ -106,26 +107,26 @@ if [[ $selection == 0 ]]; then
 
 # ###
 elif [[ $selection == 1 ]]; then
-	syncHead
+  syncHead
   # prompt user for target project
   destProj
   echo "You selected:" $userProj
 
   deploy_cmd="${SCRIPT_PATH}/git-push.sh -i ${userProj}"
-	echo "[0] Upload config.yml"
-	echo "[1] Download uploads"
-	printf "Selection: "
-	read selection2
-	if [[ $selection2 == 0 ]]; then
-		source $SCRIPT_PATH/rsync-config.sh $userProj
-	elif [[ $selection2 == 1 ]]; then
-		source $SCRIPT_PATH/rsync-uploads.sh $userProj
-	fi
+  echo "[0] Upload config.yml"
+  echo "[1] Download uploads"
+  printf "Selection: "
+  read selection2
+  if [[ $selection2 == 0 ]]; then
+    source $SCRIPT_PATH/rsync-config.sh $userProj
```

Comment:
I realize in this script we are using an .env file that lists all of the environments. I wonder if it's possible to use the list of git remote names, as the git push script relies on.

Reply:
Oh totally. We can prompt the user the same we way we do for the deploy selection. Haven't gotten to that task yet, but will likely do that before the next pre-production deployment. Thanks!